### PR TITLE
update badges

### PR DESCRIPTION
**Pull request by 0xMuluh**
improve badges. the failing bioc build is as a result of the dependence of `miaViz` on `microbiomeDatasets`. But this is resolved in our current devel.
@antagomir

### CODE REVIEW

**Comment by antagomir** [comment]
I have now updated microbiomeDataSets in Bioc devel. This should take a few days to go through. Version 1.13.3

**Comment by TuomasBorman** [comment]
Certain dataset was removwd in the summer. This is known problem, however, it does affext only the vignettes. that is why i did not do anything

But that is why those dataeets cannot be removed without deprecation warnings

21.9.2024 12.15 Muluh ***@***.***> kirjoitti:

improve badges. the failing bioc build is as a result of the dependence of miaViz on microbiomeDatasets. But this is resolved in our current devel.
@antagomir<https://github.com/antagomir>

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/microbiome/miaViz/pull/151

Commit Summary

  *   b2633e3<https://github.com/microbiome/miaViz/pull/151/commits/b2633e3fc9bb4cf48eaaccecc2717cd66d61776c> Update README.md
  *   2c0e8b9<https://github.com/microbiome/miaViz/pull/151/commits/2c0e8b995da03806d5f12eaf50c9a4e052ff1e83> Update README.md
  *   8c1acf2<https://github.com/microbiome/miaViz/pull/151/commits/8c1acf2d897eac984353c7790f126f12d6fa2263> Update README.md

File Changes

(1 file<https://github.com/microbiome/miaViz/pull/151/files>)

  *   M README.md<https://github.com/microbiome/miaViz/pull/151/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5> (7)

Patch Links:

  *   https://github.com/microbiome/miaViz/pull/151.patch
  *   https://github.com/microbiome/miaViz/pull/151.diff

**Review by antagomir — APPROVED** [review]

**Comment by antagomir** [comment]
If @Daenarys8 can fix vignette if necessary that wuld be great

**Comment by TuomasBorman** [comment]
It is already fixed, I believe. The failing happens in release of Bioc if I remember correctly. I thought that it is better to keep untouched (of course we can push to that branch also but it is easier to not). New release is in October which will solve this problem

21.9.2024 13.42 Leo Lahti ***@***.***> kirjoitti:

If @Daenarys8<https://github.com/Daenarys8> can fix vignette if necessary that wuld be great

File: README.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 <!-- badges: start -->
-
-[![R-CMD-check-Bioc-devel](https://github.com/microbiome/miaViz/workflows/R-CMD-check-bioc-devel/badge.svg)](https://github.com/microbiome/miaViz/actions)
-[![R-CMD-check-bioc](https://github.com/microbiome/mia/actions/workflows/check-bioc.yml/badge.svg)](https://github.com/microbiome/mia/actions/workflows/check-bioc.yml)
+[![Platforms](http://bioconductor.org/shields/availability/release/miaViz.svg)](https://bioconductor.org/packages/release/bioc/html/miaViz.html#archives)
+[![rworkflows](https://github.com/microbiome/miaViz/workflows/rworkflows/badge.svg)](https://github.com/microbiome/miaViz/actions)
 [![Bioc-release](http://bioconductor.org/shields/build/release/bioc/miaViz.svg)](http://bioconductor.org/packages/release/bioc/html/miaViz.html)
+[![Bioc-age](http://bioconductor.org/shields/years-in-bioc/miaViz.svg)](https://bioconductor.org/packages/release/bioc/html/miaViz.html#since)
 [![Codecov test
 coverage](https://codecov.io/gh/microbiome/miaViz/branch/master/graph/badge.svg)](https://codecov.io/gh/microbiome/miaViz?branch=master)
```
Comment (TuomasBorman) [inline README.md]: Can you switch to devel branch

File: README.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 # miaViz <img src="man/figures/mia_logo.png" align="right" width="120" />
 
 <!-- badges: start -->
-
-[![R-CMD-check-Bioc-devel](https://github.com/microbiome/miaViz/workflows/R-CMD-check-bioc-devel/badge.svg)](https://github.com/microbiome/miaViz/actions)
-[![R-CMD-check-bioc](https://github.com/microbiome/mia/actions/workflows/check-bioc.yml/badge.svg)](https://github.com/microbiome/mia/actions/workflows/check-bioc.yml)
+[![Platforms](http://bioconductor.org/shields/availability/release/miaViz.svg)](https://bioconductor.org/packages/release/bioc/html/miaViz.html#archives)
+[![rworkflows](https://github.com/microbiome/miaViz/workflows/rworkflows/badge.svg)](https://github.com/microbiome/miaViz/actions)
```
Comment (TuomasBorman) [inline README.md]: Check if you can only show actions related to devel branch

**Pull request merged**